samples/02_power_users_developers/openstreetmap_exploration.ipynb

```diff
@@ -0,0 +1,1338 @@
+{
```

Comment:
<p>I think it might clarify to use one cell to display the data and a separate cell to assign the variables. Creating the <em>full_extent </em>variable and then using <em>project() </em>on it, I expected the output to be in lat/long values because of the <em>out_sr=4326 </em>parameter.  I  read through the code again to discover that I wasn't actually displaying the values for any of those variables I created after projecting but instead was displaying the spatial dataframe that I had downloaded.  It might be helpful to explicitly state that the <em>gen_osm_df() </em>method expects a parameter in a specific format to explain why you are creating these variables.</p><p> </p><ul><li>NOTE: The narrative guides me that I could use <em>dc_df_spatial_project() </em>if I have an ArcPy enviroment. When I use this function, the output in the SHAPE column actually displays the values in lat/long coordinates, not Web Mercator coordinates.  It could be helpful to clarify whether the dataframe needs to be projected or not depending upon the environment</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>